src/uwsgi.ini

```diff
@@ -0,0 +1,23 @@
+[uwsgi]
+strict = true
+master = true
+enable-threads = true
+vacuum = true
+need-app = true
+
+processes = 4
+threads = 2
+
+py-autoreload = 1
+
+# add an http router/server on the specified address
+# see https://uwsgi-docs.readthedocs.io/en/latest/Options.html#http
+http = :8000
```

Comment:
Let op dat je poort nu anders is (was 8001). Hoeft overigens geen issue te zijn

Reply:
Dat is alleen in de docker-compose, welke niet wordt gebruikt voor de deployment.